packages/terra-clinical-label-value-view/lib/LabelValueView.scss

```diff
@@ -0,0 +1,17 @@
+@import '~terra-mixins';
+
+.terraClinical-LabelValueView {
+  border: 0;
+  padding: 0.1785rem;
+}
+
+.terraClinical-LabelValueView-label {
+  color: $terra-grey-50;
+  font-size: 1rem;
+  line-height: 20px;
+}
+
+.terraClinical-LabelValueView-value {
+  font-size: 1.4285714285714286rem;
+  line-height: 1.4285714285714286rem;
```

Comment:
I'm cool with Tyler's suggestion.  I definitely would like to see this addressed before it gets merged in.